client/components/site-selector/index.jsx

```diff
@@ -263,7 +293,7 @@ export default React.createClass( {
 					/>
 					<div className="site-selector__sites" ref="selector">
 						{ this.renderAllSites() }
-						{ this.renderRecentSites() }
+						{ this.renderStarredSites() }
```

Comment:
Should starred sites always appear first in the list? I'm not always seeing this be the case:

![image](https://cloud.githubusercontent.com/assets/1779930/12796130/46dfd7ce-ca8b-11e5-9a6a-de52fe0a0d59.png)